### DISCUSSION TITLE
The future of Eleventy

### DISCUSSION THREAD

**Discussion by bsastregx** [11ty/eleventy#3091] [Uncategorized]
Hello everybody. We are considering using Eleventy or Next.js for our new website. I totally prefer Eleventy for this case, because it is in the end, a static website, not an application, but I must wonder and take into consideration the probability of support and maintenance in the future (about 10 years), which in the case of Next.Js I would say it is high. What is the expectancy for Eleventy? Thanks in advance.

**Comment by wommy** [comment]
i feel like this [video](https://www.youtube.com/watch?v=bPtQmsjXMuo) does a decent job of addressing this
